component-sets/default-components/components-definition.json

```
@@ -26,6 +26,25 @@
             "label": { "key": "COMPONENT_PRODUCT_LABEL" },
             "properties": ["position", "style", "text-align", "letter-spacing", "line-height"]
         },
+        {
+            "name": "chart",
+            "icon": "icons/components/chart.svg",
```

I would add a handle to select the component, once iframe is placed then it's really hard to select the item 
```suggestion
            "icon": "icons/components/chart.svg",
            "selectionMethod": "handle",
```

---

💯